Mathlib/Combinatorics/Additive/Energy.lean

```diff
@@ -45,6 +41,17 @@ def multiplicativeEnergy (s t : Finset α) : ℕ :=
 #align finset.multiplicative_energy Finset.multiplicativeEnergy
 #align finset.additive_energy Finset.additiveEnergy
 
+@[inherit_doc multiplicativeEnergy] notation3:max "Eₘ[" s ", " t "]" => multiplicativeEnergy s t
+@[inherit_doc additiveEnergy] notation3:max "E[" s ", " t "]" => additiveEnergy s t
+
+/-- The multiplicative energy of a finset `s` in a group is the number of quadruples
+`(a₁, a₂, b₁, b₂) ∈ s × s × s × s` such that `a₁ * b₁ = a₂ * b₂`. -/
+notation3:max "Eₘ[" s "]" => Finset.multiplicativeEnergy s s
+
+/-- The additive energy of two finsets `s` and `t` in a group is the number of quadruples
+`(a₁, a₂, b₁, b₂) ∈ s × s × s × s` such that `a₁ + b₁ = a₂ + b₂`. -/
+notation3:max "E[" s "]" => Finset.additiveEnergy s s
+
 @[to_additive additiveEnergy_mono]
```

Comment:
instead of writing down everywhere the `to_additive` name, could you register in the relevant file that `multiplicativeEnergy` should always be additivized to `additiveEnergy`?

Reply:
Truth is I want to rename `additiveEnergy`/`multiplicativeEnergy` to `addEnergy`/`mulEnergy`, which is why I didn't want to hardcode the translation.

Since you raised that up, I will perform the rename now.